src/main/java/org/folio/consortia/service/impl/TenantServiceImpl.java

```diff
@@ -43,6 +43,19 @@ public TenantCollection get(UUID consortiumId, Integer offset, Integer limit) {
     return result;
   }
 
+  @Override
+  public String getCentralTenantId() {
+    TenantEntity tenant = tenantRepository.findCentralTenant()
+      .orElseThrow(() -> new ResourceNotFoundException("A central tenant is not found. The central tenant must be created"));
+    return tenant.getId();
+  }
+
+  @Override
+  public TenantEntity getByTenantId(String tenantId) {
+    return tenantRepository.findById(tenantId)
```

Comment:
method was changed order to put first get method, then save update delete method